Test/Test Utilities/MockVersion2Log.swift

```diff
@@ -0,0 +1,21 @@
+import Ably.Private
+
+@objc(ARTMockVersion2Log) class MockVersion2Log: NSObject, Version2Log {
```

Comment:
Not sure whether this was intentional, but usually `@objc` has a dedicated line before class declaration.

Reply:
Didn't know that! Fixed it, thanks :)